### PR TITLE
✨ feat(web): A - Input

### PR DESCRIPTION
**Pull request by HaiGentle**

### CODE REVIEW

File: apps/web/src/components/atoms/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+
+import classNames from 'classnames';
+import './styles.scss';
+
+export interface InputProps
+  extends React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement> {
+  suffix: React.ReactNode;
+  fullWidth: boolean;
```
Comment (thaihoanganh) [inline apps/web/src/components/atoms/Input/Input.tsx:9]: `suffix` and `fullWidth` should be optional.
Example:
```tsx
interface InputProps {
 suffix?: React.ReactNode;
 fullWidth?: boolean;
}
```

File: apps/web/src/components/atoms/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+export const Input: FC<InputProps> = ({ suffix, fullWidth = false, ...otherProps }) => {
+  return (
+    <div className={classNames('cpn-input', fullWidth && 'cpn-input--full-width')}>
+      <input className="cpn-input__input" type="text" {...otherProps} />
```
Comment (thaihoanganh) [inline apps/web/src/components/atoms/Input/Input.tsx:15]: An input element has type default is `text`, you can remove this attribute.

File: apps/web/src/components/atoms/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      {suffix && <div className="cpn-input__icon">{suffix}</div>}
```
Comment (thaihoanganh) [inline apps/web/src/components/atoms/Input/Input.tsx:16]: `Input` is an `atom` element, you can change the prefix of the class name to `a-`.
Example: `a-input` `a-input__input`

File: apps/web/src/components/atoms/Input/Input.tsx (hunk 1 above)
Comment (HaiGentle) [reply apps/web/src/components/atoms/Input/Input.tsx]: updated

**Comment by thaihoanganh** [comment]
@HaiGentle I have edited the design, I will continue to do your work.

**Review by gialynguyen — APPROVED** [review]

**Pull request merged**